about.html

```diff
@@ -131,7 +133,7 @@ <h2 property="schema:name">Features</h2>
           <li><a href="/for-developers.html" rel="rdfs:seeAlso">For Developers</a></li>
           <li><a href="/for-organizations.html" rel="rdfs:seeAlso">For Organizations</a></li>
           <li><a href="/community.html" rel="rdfs:seeAlso">Community</a></li>
-          <li><a href="/#">License</a></li>
+          <li><a href="https://github.com/solid/solidproject.org/blob/main/LICENSE.md/#">License</a></li>
```

Comment:
```suggestion
          <li><a href="https://github.com/solid/solidproject.org/blob/main/LICENSE.md">License</a></li>
```

We should probably use CC0 or CC-BY for the website content. Perhaps create an issue for consideration out of this?

Reply:
I agree, but let's create an issue for it